plugin/v2/settable.go

```diff
@@ -0,0 +1,73 @@
+package v2
+
+import (
+	"fmt"
+	"strings"
+)
+
+type settable struct {
+	name  string
+	field string
```

Comment:
it's the name of field you want to set as a string, most of the time it will be "value"